lib/systemd/system/revpi-bthelper@.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Revolution Pi bluetooth helper for %I
```

Comment:
Is `%I` valid in the description?

Reply:
yes

pi@RevPi12345:~ $ systemctl status revpi-bthelper@hci0.service
● revpi-bthelper@hci0.service - Revolution Pi bluetooth helper for **hci0**
   Loaded: loaded (/lib/systemd/system/revpi-bthelper@.service; static; vendor preset: enabled)
   Active: active (exited) since Tue 2021-03-16 15:30:09 CET; 44min ago
  Process: 3447 ExecStart=/usr/sbin/revpi-bthelper hci0 (code=exited, status=0/SUCCESS)
 Main PID: 3447 (code=exited, status=0/SUCCESS)

Mar 16 15:30:09 RevPi12345 systemd[1]: Starting Revolution Pi bluetooth helper for **hci0**...
Mar 16 15:30:09 RevPi12345 revpi-bthelper[3447]: Not UART-attached BT Modem on Revolution Pi
Mar 16 15:30:09 RevPi12345 systemd[1]: Started Revolution Pi bluetooth helper for **hci0**.


pi@RevPi12345:~ $ systemctl status revpi-hciuart@ttyUSB0.service
● revpi-hciuart@ttyUSB0.service - Revolution Pi Bluetooth Device **ttyUSB0**
   Loaded: loaded (/lib/systemd/system/revpi-hciuart@.service; static; vendor preset: enabled)
   Active: active (running) since Tue 2021-03-16 15:30:09 CET; 42min ago
  Process: 2958 ExecStart=/usr/bin/revpi-btuart ttyUSB0 (code=exited, status=0/SUCCESS)
 Main PID: 3402 (hciattach)
    Tasks: 1 (limit: 2199)
   Memory: 568.0K
   CGroup: /system.slice/system-revpi\x2dhciuart.slice/revpi-hciuart@ttyUSB0.service
           └─3402 /usr/bin/hciattach ttyUSB0 bcm43xx 2000000 flow

Mar 16 15:29:58 RevPi12345 systemd[1]: Starting Revolution Pi Bluetooth Device **ttyUSB0**...
Mar 16 15:30:09 RevPi12345 revpi-btuart[2958]: bcm43xx_init
Mar 16 15:30:09 RevPi12345 revpi-btuart[2958]: Flash firmware /lib/firmware/brcm/BCM4345C0.hcd
Mar 16 15:30:09 RevPi12345 revpi-btuart[2958]: Set Controller UART speed to 2000000 bit/s
Mar 16 15:30:09 RevPi12345 revpi-btuart[2958]: Device setup complete
Mar 16 15:30:09 RevPi12345 systemd[1]: Started Revolution Pi Bluetooth Device **ttyUSB0**.
pi@RevPi12345:~ $